src/extension.ts

```diff
@@ -158,18 +158,20 @@ function interpolateVSCodeSpecificExecutablePath(executablePath: string): string
 //
 // If it fails either of these checks, a status is returned to show that the
 // executable path is not valid.
-async function getWaspExecutableVersion(executablePath: string): Promise<ExecutableErrorStatus | string> {
+async function getWaspExecutableVersion(executablePath: string): Promise<WaspVersionError | string> {
   const execFileP = promisify(execFile);
   const execPromise = execFileP(executablePath, ['version'], { timeout: 2000, windowsHide: true })
-    .then(({ stdout }) => stdout)
-    .catch(() => ExecutableErrorStatus.Missing);
+    .then(({ stdout }) => stdout.split("\n")[0])  // We assume `wasp version` returns version as a single string in the first line, with nothing else.
```

Comment:
We had a bug where we thought the whole output of `wasp version` is semver, which it was before, but it is not anymore because we now also output some instructions, so I modified this to take only the first line, as a relatively quick fix.

In the future we will probably want to have version (hoho) of `wasp version` that takes flag and guarantees always the same output, smth like `wasp version --semver` or `wasp version --number`.